main.go

```diff
@@ -134,23 +138,65 @@ func updateNDK(revision string) error {
 		return err
 	}
 
+	if err := updateNDKPathIfNeeded(ndkHome); err != nil {
```

Comment:
instead of ifneeded we use ensure, for example ensureNDKPath (to keep it consistent)

Reply:
Renamed.